src/kestrel_datasource_stixshifter/interface.py

```diff
@@ -195,7 +192,9 @@ def query(uri, pattern, session_id, config, store=None):
             check_module_availability(connector_name)
 
             data_path_striped = "".join(filter(str.isalnum, profile))
-            ingestfile = ingestdir / f"{i}_{data_path_striped}.json"
+            ingestfile = ingestdir / f"{i}_batch_index_{data_path_striped}.json"
+
+            ingest_stixbundle_filepath = make_ingest_stixbundle_filepath(ingestfile)
```

Comment:
Could we combine the two lines into the `make_ingest_stixbundle_filepath()` function:
```
ingestfile = ingestdir / f"{i}_batch_index_{data_path_striped}.json"
ingest_stixbundle_filepath = make_ingest_stixbundle_filepath(ingestfile)
```
and not use `replace()`?